Lib/pdb.py

```diff
@@ -1315,7 +1315,12 @@ def do_commands(self, arg):
         reached.
         """
         if not arg:
-            bnum = len(bdb.Breakpoint.bpbynumber) - 1
+            for bnum in range(len(bdb.Breakpoint.bpbynumber) - 1, -1, -1):
```

Comment:
I'm not sure, but it looks like using `reversed(range(...))` instead of `range(..., -1, -1)`  would be more readable and make the reverse iteration intent clearer, what do you think?

```suggestion
            for bnum in reversed(range(len(bdb.Breakpoint.bpbynumber))):
```

Reply:
See also https://github.com/python/cpython/pull/142835#discussion_r2623943070.